cmake/modules/FindNFS.cmake

```
@@ -34,7 +34,26 @@ find_package_handle_standard_args(NFS
 if(NFS_FOUND)
   set(NFS_LIBRARIES ${NFS_LIBRARY})
   set(NFS_INCLUDE_DIRS ${NFS_INCLUDE_DIR})
+  if(WIN32)
+    list(APPEND NFS_LIBRARIES ws2_32.lib)
+  endif()
+
+  set(CMAKE_REQUIRED_INCLUDES ${NFS_INCLUDE_DIRS})
+  set(CMAKE_REQUIRED_LIBRARIES ${NFS_LIBRARIES})
+
+  set(CHECK_FILES nfsc/libnfs.h)
+  if(WIN32)
+    list(INSERT CHECK_FILES 0 Winsock2.h)
```

On my Linux system I had to do the following adjustment to succeed the symbol check with libnfs 3.0.0.
```suggestion
    list(INSERT CHECK_FILES 0 Winsock2.h)
  else()
    list(INSERT CHECK_FILES 0 sys/time.h)
```
\
\
Those are the errors from cmake compile testing.
```
/usr/include/nfsc/libnfs.h:1181:23: error: field ‘atime’ has incomplete type
 1181 |        struct timeval atime;
      |                       ^~~~~
/usr/include/nfsc/libnfs.h:1182:23: error: field ‘mtime’ has incomplete type
 1182 |        struct timeval mtime;
      |                       ^~~~~
/usr/include/nfsc/libnfs.h:1183:23: error: field ‘ctime’ has incomplete type
 1183 |        struct timeval ctime;
      |                       ^~~~~
```

---

Jenkins behaves the same on all Linux targets and FreeBSD.